src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx

```diff
@@ -113,3 +112,20 @@ export const AuctionArtworksRailFragmentContainer = createFragmentContainer(
     `,
   }
 )
+
+const tracks = {
+  clickedArtworkGroupHeader: (
+    contextModule: ContextModule,
+    contextPageOwnerType: PageOwnerType,
+    saleID: string,
+    saleSlug: string
+  ): ClickedArtworkGroup => ({
```

Comment:
this should either be removed or defined in force... or?

Reply:
You mean the schema type? That's here https://github.com/artsy/cohesion/blob/d9f09f800282a372e2e182a407b42ee153181491/src/Schema/Events/Click.ts#L164.